app/Migrations/Version20211214113051.php

```diff
@@ -53,7 +53,8 @@ public function up(Schema $schema): void
         $this->addSql('CREATE INDEX IDX_613760068DB60186 ON sdk_tasks_placeholders (task_id)');
         $this->addSql('CREATE UNIQUE INDEX UNIQ_61376006DA75C033 ON sdk_tasks_placeholders (placeholder_id)');
         $this->addSql('CREATE TABLE sdk_workflow (id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, project VARCHAR(255) NOT NULL COLLATE BINARY, status CLOB NOT NULL --(DC2Type:json)
-        , workflow VARCHAR(255) NOT NULL COLLATE BINARY)');
+        , workflow VARCHAR(255) NOT NULL COLLATE BINARY, data CLOB NOT NULL --(DC2Type:json)
```

Comment:
data-> new table id, workflowId, transactionName, data for history
path -> int parent nullable

Reply:
done